frontend/css/cart.css

```diff
@@ -0,0 +1,95 @@
+.cart {
+    position: fixed;
+    top: 0;
+    right: 0;
+    display: none;
+    width: 360px;
+    height: 100vh;
+    overflow-y: auto;
+    overflow-x: hidden;
+    padding: var(--spacing-m);
+    background-color: var(--main-background-color);
+}
+
+.cart-title {
+    text-align: center;
+    font-size: var(--h2-font-size);
+    margin-top: var(--spacing-l);
+}
+
+.cart-box {
+    display: grid;
+    grid-template-columns: 1.5fr 2fr 0.2fr;
+    align-items: center;
+    gap: var(--spacing-m);
+    margin-top: var(--spacing-m);
+}
+
+.cart-img {
+    width: 100%;
```

Comment:
if you don't specify 100%, what do you get (is 100% not the default)?  is this needed to override some inherited width styling?

Reply:
Without the 100%, the image of the products overflows the container